packages/google_sign_in/lib/google_sign_in_tizen.dart

```diff
@@ -47,24 +46,23 @@ class _GoogleSignInTokenDataTizen extends GoogleSignInTokenData {
   }
 
   /// Creates a [_GoogleSignInTokenDataTizen] from a json object.
-  static _GoogleSignInTokenDataTizen fromJson(Map<String, Object?> json) {
+  static _GoogleSignInTokenDataTizen fromJson(Map<String, dynamic> json) {
```

Comment:
`Object` or `Object?` is preferred over `dynamic`, since `dynamic` disables all static analysis.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic

Reply:
Thank you :)